tests/trainer/resources/osft.ipynb

```
@@ -158,80 +173,53 @@
-    "                                pass\n",
-    "\n",
-    "        if pulled_any:\n",
-    "            print(f\"[notebook] ✓ S3 download successful. Processed {file_count} files\")\n",
+    "                    dst = os.path.join(local_dir, rel)\n",
+    "                    os.makedirs(os.path.dirname(dst), exist_ok=True)\n",
+    "                    \n",
+    "                    if not os.path.exists(dst):\n",
+    "                        if stream_download(s3_client, bucket, key, dst):\n",
+    "                            count += 1\n",
+    "                            # Decompress .gz files\n",
+    "                            if dst.endswith(\".gz\"):\n",
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)